spec/onchain.tex

```diff
@@ -414,19 +475,60 @@ \subsection{Close Transaction}\label{sec:close-tx}
 		\[
 			\eta' = \eta
 		\]
-		\item $\mathsf{Current}$: Closing snapshot refers to current state version $v$ and any UTxO to decommit need to be present in the closed state too.
+    \red{
+		  \item $\mathsf{UnusedInc}$: Closing snapshot refers to current state version $v$ and any UTxO to increment must be $\bot$ in the closed state.
+		  \[
+		  	\eta_\alpha' = \eta_\alpha = \eta_\Delta' = \bot
+		  \]
+		  \[
+		  	\eta_\omega' = \bot
+		  \]
+		  \[
+		  \msVfy(\hydraKeys,(\cid || v || s' || \eta' || \eta_\alpha' || \eta_\omega'),\xi) = \true
+		  \]
+      which implies $\eta_\Delta' = \eta_\alpha$ while $\eta_\alpha$ is provided by the redeemer \footnote{$\eta_\alpha$ needs to be provided to verify the signature, but is otherwise not relevant for the closed state}.
+    }
+		\item $\mathsf{UnusedDec}$: Closing snapshot refers to current state version $v$ and any UTxO to decrement need to be present in the closed state too.
+		\[
+			\eta_\omega' = \eta_\omega = \eta_\Delta' \neq \bot
+		\]
+    \red{
+		  \[
+		  	\eta_\alpha' = \bot
+		  \]
+    }
 		\[
-			\msVfy(\hydraKeys,(\cid || v || s' || \eta' || \eta_{\Delta}'),\xi) = \true
+      \msVfy(\hydraKeys,(\cid || v || s' || \eta' \red{|| \eta_\alpha'} || \eta_\omega'),\xi) = \true
 		\]
-		which implies $\eta_{\Delta}' = \eta_{\omega}$ while $\eta_{\omega}$ does not need to be provided.
-		\item $\mathsf{Outdated}$: Closing snapshot refers the previous state $v - 1$ and any UTxO to decommit of the closing snapshot must not be recorded in the closed state.
+    which implies $\eta_\Delta' = \eta_\omega$.
+		% TODO: put more into the redeemer to simplify?
+		\todo{this is hard to understand}
+    \red{
+		  \item $\mathsf{UsedInc}$: Closing snapshot refers the previous state $v - 1$ and any UTxO to increment of the closing snapshot must be recorded in the closed state.
+		  \[
+		  	\eta_\alpha' = \eta_\alpha = \eta_\Delta' \neq \bot
+		  \]
+		  \[
+		  	\eta_\omega' = \bot
+		  \]
+		  \[
+		  	\msVfy(\hydraKeys,(\cid || v - 1 || s' || \eta' || \eta_\alpha' || \eta_\omega' ),\xi) = \true
+		  \]
+      which implies $\eta_\Delta' = \eta_\alpha$.
+    }
+		\item $\mathsf{UsedDec}$: Closing snapshot refers the previous state $v - 1$ and any UTxO to decrement of the closing snapshot must not be recorded in the closed state.
 		\[
-			\eta_{\Delta}' = \bot
+	 		\eta_\omega' = \eta_\omega = \eta_\Delta' \neq \bot
```

Comment:
This is wrong!

In the diff you can clearly see that we explicitly required $\eta_{\Delta}' = \bot$ before and this is still the case (the english sentence above also claims that).